common/rya.api/src/main/java/mvm/rya/api/persist/index/RyaSecondaryIndexer.java

```diff
@@ -34,7 +34,7 @@
 
 public interface RyaSecondaryIndexer extends Closeable, Flushable, Configurable {
 
-    public String getTableName();
+    public String getTableName(); // TODO: some indexers have multiple tables, should return a list.
```

Comment:
Actually, we should remove this method entirely in future versions of the application. A SecondaryIndexer's storage mechanism shouldn't be assumed to be a database table.